--- lib/class/song.class.php.orig	2013-12-02 13:28:35.850000000 +0100
+++ lib/class/song.class.php	2013-12-02 13:29:24.190000000 +0100
@@ -313,6 +313,9 @@
             case 'mpc':
                 return 'audio/x-musepack';
             break;
+            case 'aacp':
+                return 'audio/aacp';
+            break;
             default:
                 return 'audio/mpeg';
             break;
